projects/incubator/src/storefrontlib/incubator-storefrontlib.module.ts

```diff
@@ -0,0 +1,4 @@
+import { NgModule } from '@angular/core';
+
+@NgModule({})
+export class IncubatorStorefrontlibModule {}
```

Comment:
Great that you added the 2 modules. Can we stick to `IncubatorStorefrontModule` as I like to avoid `lib` in the name (lib is currently only in internal references).